doc/howto/usage/k8s/k8s_aws_en.md

```diff
@@ -364,6 +376,26 @@ paddle-cl-ElbAPISe-EEOI3EZPR86C-531251350.us-west-2.elb.amazonaws.com. 59 IN A 5
 
 In the above output, both ip `54.241.164.52`, `54.67.102.112` will work.
 
+*If you own a DNS name*, set the A record to any of the above ip. Then you can skip to the step "Access the cluster".
+
+*If you do not own a DNS name*:
+##### Update local DNS association
+Edit `/etc/hosts` to associate above ip with the DNS name.
+##### Add Route53 private name service in VPC
+ - Open Route53 web console
```

Comment:
sure, will add link to the name. also, we might need to add some explain of the instructions, rather than just "steps of operation".